oi/TreeBuilder.cpp

```diff
@@ -379,8 +379,7 @@ uint64_t TreeBuilder::next() {
 
 bool TreeBuilder::isContainer(const Variable& variable) {
   return th->containerTypeMap.contains(variable.type) ||
-         (drgn_type_kind(variable.type) == DRGN_TYPE_ARRAY &&
-          drgn_type_length(variable.type) > 0);
+         drgn_type_kind(variable.type) == DRGN_TYPE_ARRAY;
```

Comment:
This change breaks CodeGen-v1 for zero length arrays: `make oid && OID_TEST_ARGS='-Ftype-graph' build/test/integration/integration_test_runner --verbose --preserve --force --gtest_filter="OidIntegration.arrays_member_int0"`

I think this works instead:
```cpp
bool TreeBuilder::isContainer(const Variable& variable) {
  if (th->containerTypeMap.contains(variable.type))
    return true;
  if (drgn_type_kind(variable.type) != DRGN_TYPE_ARRAY)
    return false;
  return config.features[Feature::TypeGraph] || drgn_type_length(variable.type) > 0;
}
```